test/Governance.test.ts

```diff
@@ -93,51 +93,96 @@ describe('Governance', function () {
     )
     await this.rewards.deployed()
     await this.governance.setInitialTreasury(this.treasury.address)
+
     this.votingPower = BN.from(VOTING_POWER)
     await this.governance.setVoterRolesAdmin(this.delegatee1.address, [
       DEVELOPER_ROLE
     ])
+    expect(
```

Comment:
question: why verify the state in the setup not the tests?

Usual structure would be verifying the state in the test, unless you're needing to verify the setup operations worked correctly?

Reply:
Will make separate test for checking initial state.